src/codeu/chat/client/commandline/Chat.java

```diff
@@ -181,6 +181,24 @@ private UserContext findUser(String name) {
       }
     });
 
+      /**
+       * Command line command that will use the context that was created
+       */
+    panel.register("info", new Panel.Command(){
+      @Override
+      public void invoke(Scanner args){
+        final UptimeServerInfo info = context.getInfo();
+        if(info == null){
+          // Communicate error to user - the server did not send a valid info object.
+          System.out.println("The server did not send a valid info object.");
+        }
+        else{
+          //THIS NEEDS TO BE UPDATED
+          System.out.println("RUNNNING INFO");
+        }
+      }
+    });
+
```

Comment:
This method only displays the UpTimeServerInfo information. It also needs to display the VersionCheckServerInfo information.